Week1/homework/app.js

```diff
@@ -1,11 +1,187 @@
-'use strict';
+/* eslint-disable no-unused-expressions */
+{
+  ('use strict');
+  let bookTitles = [
+    'deep_work',
+    'anna_frank',
+    'forty_rules',
+    'memory_in_flesh',
+    'dust_of_promises',
+    'art_of_forgetting',
+    'black_suits_you',
+    'Chaos_Of_Senses',
+    'bed_hopper',
+    'game_of_thrones',
+  ];
+  console.log(bookTitles);
+
+  const listElement = document.createElement('ul');
+  function listElements(ul, itemsArray) {
```

Comment:
You don't use this function anymore, remember to clean up your code when you make a pull request.